src/EditorFactory.js

```diff
@@ -65,8 +63,7 @@ const createPlainEditor = ({ language, extensions = [] } = {}) => {
 	return new Editor({
 		editorProps,
 		extensions: [
-			PlainText,
-			CodeBlockLowlight.configure({
+			PlainText.configure({
```

Comment:
```suggestion
			PlainText.configure({
					// Options to be passed through to `CodeBlockPlainText`
```

How about a short code comment here  to explain why we pass codeblock config options to the plaintext extension?